### PR TITLE
feat: add network suffix to payment url

### LINKED ISSUE #460: Payment Permit URL Must Specify Network ID Now
We now pass in the network ID as a URL query parameter so that the bounty claims dApp (pay.ubq.fi) can signal to the user to switch to the correct network.

The new URL structure is as follows:

`pay.ubq.fi?claim=${base64permit}&network=${networkId}`

> > the query parameter is `chainId` so for gnosis chain it is: [permit](https://4bbbf7fb.pay-ubq-fi-bxp.pages.dev/?claim=eyJwZXJtaXQiOnsicGVybWl0dGVkIjp7InRva2VuIjoiMHhlOTFEMTUzRTBiNDE1MThBMkNlOERkM0Q3OTQ0RmE4NjM0NjNhOTdkIiwiYW1vdW50IjoiMTAwMDAwMDAwMDAwMDAwMDAwMDAwMCJ9LCJub25jZSI6IjQ4MjcxMDY4ODQ1NjIyMTMwNzc3MTY4MzA3Nzg4OTg0NTY1NzkyNDcyODQ3NDE5NjUxMjYzOTAwMjUyNzA5MzA1NjY4Mzk0MzcxNjA4IiwiZGVhZGxpbmUiOiIxMTU3OTIwODkyMzczMTYxOTU0MjM1NzA5ODUwMDg2ODc5MDc4NTMyNjk5ODQ2NjU2NDA1NjQwMzk0NTc1ODQwMDc5MTMxMjk2Mzk5MzUifSwidHJhbnNmZXJEZXRhaWxzIjp7InRvIjoiMHg0MDA3Q0UyMDgzYzdGM0UxODA5N2FlQjNBMzliYjhlQzE0OWEzNDFkIiwicmVxdWVzdGVkQW1vdW50IjoiMTAwMDAwMDAwMDAwMDAwMDAwMDAwMCJ9LCJvd25lciI6IjB4Zjg3Y2E0NTgzQzc5MjIxMmU1MjcyMGQxMjdFN0UwQTM4QjgxOGFEMSIsInNpZ25hdHVyZSI6IjB4ZWRiZTg1ZTAxYTc0Yjc5MjRjY2Y1ZTMzMzllN2U3NWVmMDJlZGMwMDI0N2YxZTk5YzZhOThkY2U4YzA3MmZiZDM3NTU0M2NkMWZhYmJkNDA5Y2JhNWRkMGZhYThkZWRhYTA2NjY2MzI1YzMxNjM4ZDQ1ZTcwYTQ4YTQ0NDY2ZGIxYiJ9&chainId=0x64)
> 
> This is looking fantastic. Thanks! 
> 
> Please be sure to resolve all of the conversation points before we merge this, including renaming all instances of "chain" to "network" https://github.com/ubiquity/pay.ubq.fi/pull/76/files#r1242357293
> 
> Can you rename the query parameter to `?network=100` instead of using hexadecimal?
> 
> I also want to give a small bonus when this is completed for helping us debug the deployment issue!

_Originally posted by @pavlovcik in https://github.com/ubiquity/pay.ubq.fi/issues/76#issuecomment-1621044544_

### PR DESCRIPTION
**Pull request by devpanther**
Resolves #460

### CODE REVIEW

File: src/helpers/permit.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const generatePermit2Signature = async (spender: string, amountInEth: str
 
   const base64encodedTxData = Buffer.from(JSON.stringify(txData)).toString("base64");
 
-  const result = `${permitBaseUrl}?claim=${base64encodedTxData}&chainId=0x${chainId.toString(16)}`;
+  const result = `${permitBaseUrl}?claim=${base64encodedTxData}&network=${chainId.toString(10)}`;
```
Comment (pavlovcik) [inline src/helpers/permit.ts:60]: Do you need to specify base 10?

Comment (devpanther) [reply src/helpers/permit.ts:60]: I don't need to do that

Comment (pavlovcik) [reply src/helpers/permit.ts:60]: You should remove that code if it isn't necessary.

Comment (devpanther) [reply src/helpers/permit.ts]: Removed

**Review by pavlovcik — APPROVED** [review]

**Pull request merged**